wp-content/themes/core/assets/js/src/theme/config/wp-settings.js

```diff
@@ -1,5 +1,6 @@
-
-const wp = window.modern_tribe_config;
+// It's possible our context is the block editor in the admin theme, so fall
+// back to that config.
+const wp = window.modern_tribe_config || window.modern_tribe_admin_config || {};
```

Comment:
Turns out `modern_tribe_config` isn't set in the admin theme/block editor context. Here we fall back to the `modern_tribe_admin_config` instead, but this isn't a very sophisticated way of handling this issue. Thoughts, @nickpelton and @dpellenwood?

Reply:
I'm sure there will be some interesting issues to catch here, because component JS could be running in either the frontend **or** admin theme contexts now. I'm not 100% sure the tests/tools/config JS utilities will "just work" in both contexts, but—since the slider is the component doing this at the moment—everything currently used in both contexts appears to work.